res/database/Default/user/scr0llbaer.xml

```diff
@@ -0,0 +1,1586 @@
+					<title>
+						<de>Das Elixier</de>
+						<en>The Elixir</en>
+					</title>
+					<description>
+						<de>Evelyn kann sich schon ganz gut selbst helfen. Und Raymond verliert auch noch das Elixier bei seiner tollpatschigen Rettungsaktion. Wie kann er nun im großen Showdown LeCruck doch noch besiegen?</de>
+						<en>Evelyn can already help herself quite well. And Raymond also loses the elixir during his clumsy rescue attempt. How can he still defeat LeCruck in the big showdown?</en>
+					</description>
+				</scripttemplate>
+			</children>
+
+			<jobs>
+				<job index="0" function="1" required="1" />
+				<job index="1" function="2" required="1" gender="1" role_guid="6e1f331a-7c9d-433b-bc9f-7606ad322a5e" />
+				<job index="2" function="2" required="1" gender="2" role_guid="3ee1d44a-f3fc-47a5-aabf-25803e6e9f99" />
+				<job index="3" function="2" required="1" gender="1" role_guid="410a932c-9ceb-4080-9dbc-f82ac3c9bcd3" />
+				<job index="4" function="2" required="1" gender="2" />
+				<job index="5" function="2" required="1" gender="1" />
+				<job index="6" function="2" required="1" gender="2" />
+				<job index="7" function="2" required="1" gender="1" />
+			</jobs>
+
+			<genres maingenre="1" subgenres="5,10" />
+			<episodes min="12" max="12" />
+			<blocks value="1" />
+			<price min="80000" max="130000" />
+			<potential min="59" max="95" />
+			<outcome min="71" max="95" />
+			<review min="79" max="95" />
+			<speed min="54" max="89" />
+			<studio_size value="3" />
+		</scripttemplate>
+
+		<scripttemplate product="2" licence_type="3" guid="61661455-9607-44fd-a090-c1caa8ee3205" creator="9551" comment="inspired from https://en.wikipedia.org/wiki/Dead_Winter_Dead">
+			<title>
+				<de>%dead% %winter% in %sarajevo%</de>
+				<en>%dead% %winter% in %sarajevo%</en>
+			</title>
+			<description>
+				<de>Kriegsdrama über den jungen, enthusiastischen und leicht beeindruckbaren Stevan auf der einen, und die warmherzige, aber resolute Kara auf der anderen Seite der Front in Jugoslawien, und einem Teufels-Cellisten dazwischen.</de>
+				<en>War drama about the young, enthusiastic and impressionable Stevan on one side, and the warm-hearted but resolute Kara on the other side of the front in Yugoslavia, and a devil's cellist in between.</en>
+			</description>
+
+			<variables>
+				<dead>
+					<de>Toter|Karger|Kalter</de>
+					<en>Dead|Desolate|Cold</en>
+				</dead>
+				<winter>
+					<de>Winter|Schnee|Schneeregen|Eisregen|Frost</de>
+					<en>Winter|Snow|Rain and Snow|Freezing Rain|Frost</en>
+				</winter>
+				<sarajevo>
+					<de>Sarajevo|Bosnien|Serbien|Jugoslawien</de>
+					<en>Sarajevo|Bosnia|Serbia|Yugoslavia</en>
+				</sarajevo>
+			</variables>
+
+			<children>
+				<scripttemplate index="0" product="2" licence_type="2" guid="971877f7-c985-45a0-84ad-9aaba6dd72a2">
+					<title>
+						<de>Sarajevo</de>
+						<en>Sarajevo</en>
+					</title>
+					<description>
```

Comment:
remove Gnom? Wasserspeier should be enough

Reply:
I think "Wasserspeier" alone doesn't imply it's in the form of a (mythological) figure (as it is intended there).